src/services/websocket_service.ts

```diff
@@ -131,9 +131,11 @@ export class WebsocketService {
         this._pongIntervalId = setInterval(this._cleanupConnections.bind(this), wsOpts.pongInterval);
         this._meshClient = meshClient;
 
-        this._orderEventsSubscription = this._meshClient
-            .onOrderEvents()
-            .subscribe(events => this.orderUpdate(meshUtils.orderInfosToApiOrders(events.map(e => e.order))));
+        this._meshClient.getStatsAsync().then(() => {
```

Comment:
**TODO**: strange hack due to a mesh client race condition. Investigate this more